lib/world.rb

```diff
@@ -50,6 +50,16 @@ def debug_in_after_hook
       end
     end
 
+    def debug_stmts_on_failures
```

Comment:
@jmekkatt perhaps change the method name to `print_debugging_info`

Reply:
Done